modules/appeals_api/app/swagger/appeals_api/v2/responses_contestable_issues.json

```diff
@@ -0,0 +1,151 @@
+                        "example": false
+                      },
+                      "latestIssuesInChain": {
+                        "type": "array",
+                        "description": "Shows the chain of decision and rating issues that preceded this issue. Only the most recent issue can be contested (the object itself that contains the latestIssuesInChain attribute).",
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "id": { "type": ["integer", "string"], "nullable": true, "example": null },
+                            "approxDecisionDate": {
+                              "type": "string",
+                              "nullable": true,
+                              "format": "date",
+                              "example": "2006-11-27"
+                            }
+                          }
+                        }
+                      },
+                      "ratingIssueSubjectText": {
+                        "type": "string",
+                        "nullable": true,
+                        "description": "Short description of RatingIssue",
+                        "example": "Hypertension"
+                      },
+                      "ratingIssuePercentNumber": {
+                        "type": "string",
+                        "nullable": true,
+                        "description": "Numerical rating for RatingIssue",
+                        "example": "10"
+                      },
+                      "isRating": { "type": "boolean", "description": "Whether or not this is a RatingIssue", "example": true }}}}}}}}}}},
+  "404": {
+    "description": "Veteran not found",
+    "content": {
+      "application/vnd.api+json": {
+        "schema": {
+          "type": "object",
+          "properties": { "errors": { "$ref": "#/components/schemas/errorWithTitleAndDetail" } }},
+        "examples": {
+          "bad file number or SSN": {
+            "value": {
+              "errors": [{ "status": 404, "code": "veteran_not_found", "title": "Veteran Not Found" }]}}}}}},
+  "422": {
+    "description": "Bad receipt date",
+    "content": {
+      "application/vnd.api+json": {
+        "schema": {
+          "type": "object",
+          "properties": { "errors": { "$ref": "#/components/schemas/errorWithTitleAndDetail" } }},
+        "examples": {
+          "before AMA": {
+            "value": {
+              "errors": [{ "status": 422, "code": "invalid_receipt_date", "title": "Invalid Receipt Date", "detail": "\"2019-02-18\" is before AMA Activation Date (2019-02-19)." }]}},
+          "future receipt date": {
+            "value": {
+              "errors": [{ "status": 422, "code": "invalid_receipt_date", "title": "Invalid Receipt Date", "detail": "\"2020-07-02\" is in the future (today: 2020-07-01; time zone: (GMT-05:00) America/New_York)." }]}},
```

Comment:
Would the "today" portion be adjusted based on the request's timezone? Is that even something we want to support? I'm not sure how useful reporting the current system time is (whether adjusted for the timezone or not), but not saying it _isn't_ useful either.

Also, remember that JSON:API ErrorObject allows for a `meta` key, so an alternative that may be more programmatically useful to the consumers might look like:
```
{...
  detail: "Receipt date is in the future", 
  meta: { 
    given_date: "2020-07-02", 
    current_date: "2020-07-17", 
    timezone: { 
      offset: "GMT -5:00", 
      name: "America/New_York" } 
}
```